modules/aerodyn/src/AeroDyn_IO.f90

```diff
@@ -344,15 +353,14 @@ subroutine Calc_WriteOutput_AD()
       end do  ! k=blades
    
          ! rotor outputs
-      if ( EqualRealNos( real(m%V_dot_x,SiKi), 0.0_SiKi ) ) then
-         m%AllOuts( RtTSR )    = 0.0_ReKi
+      if ( abs( m%V_dot_x ) < 0.04_ReKi .or. p%NumBlades == 0 ) then ! < 0.018 is close to "v_dot_x**3 not equal to 0" (cubed because of Cp denominator)
```

Comment:
It seems like `0.04` could be a parameter of AeroDyn if it's used elsewhere as a limit.

Reply:
The '0.04' m/s limit is only used in this IF statement for calculating output-to-file values for Cp, Cq, and Ct. I had originally used 0.018 m/s, which is similar (but faster) than checking that the denominator in the Cp equation is not zero, but it still left us with large instantaneous values for Cp, Cq, and Ct in some of the DLCs with large wind direction changes.